hack/make/.ensure-syscall-test

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+# Build a C binary for cloning a userns for seccomp tests
+# and compile it for target daemon
+(
+	if [ "$(go env GOOS)" = "linux" ]; then
+		cd contrib/syscall-test
+		docker build -qt syscall-test . > /dev/null
```

Comment:
Oh, and you could just do `docker build -qt syscall-test contrib/syscall-test` and skip the `cd` and thus subshell entirely. :innocent: